app/commands/robles_cli.rb

```diff
@@ -30,6 +30,29 @@ def lint
     exit 1 unless output.validated
   end
 
+  desc 'secrets [REPO]', 'configures a book repo with the necessary secrets'
+  long_desc <<-LONGDESC
+    `robles secrets [REPO]` will upload the secrets requires to run robles on a
+    git repository containing a book.
+
+    You must ensure that the required secrets are provided as robleenvironment variables
```

Comment:
I think there is a typo here: "robleenvironment"

Reply:
Thanks!